parcel_test.go

```diff
@@ -34,26 +34,22 @@ func TestAddGetDelete(t *testing.T) {
     parcel := getTestParcel()
 
     number, err := store.Add(parcel)
-    if err != nil {
         require.NoError(t, err)
-    }
-    assert.NotEmpty(t, number)
+
 
     addedParcel, err := store.Get(number)
-    if err != nil {
         require.NoError(t, err)
-    }
 
-    assert.EqualValues(t, parcel.Client, addedParcel.Client)
-    assert.EqualValues(t, parcel.Status, addedParcel.Status)
-    assert.EqualValues(t, parcel.Address, addedParcel.Address)
-    assert.EqualValues(t, parcel.CreatedAt, addedParcel.CreatedAt)
+    assert.Equal(t, parcel.Client, addedParcel.Client)
+    assert.Equal(t, parcel.Status, addedParcel.Status)
+    assert.Equal(t, parcel.Address, addedParcel.Address)
+    assert.Equal(t, parcel.CreatedAt, addedParcel.CreatedAt)
 
     err = store.Delete(number)
     require.NoError(t, err)
 
     _, err = store.Get(number)
-    require.Equal(t, sql.ErrNoRows, err)
+    require.ErrorIs(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```